person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :age, :name
+
+  # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
* I suggest you try to all fix rubocop error instead of disabling it as rubocop helps you to write clean code